app/assets/stylesheets/layout.scss

```diff
@@ -895,3 +895,23 @@ hr {
 .flash.flash-error p:first-of-type {
   margin-top: 0;
 }
+
+dl.form.errored.primer-new {
```

Comment:
@nwoodthorpe Could we just override the original classes?

That way when we look at the documentation we don't have to go back and remember that we made a new class?

Could we also move this into the [`forms.scss`](https://github.com/education/classroom/blob/1c9b7e1f23d2f8b4a010d1cb820749b1a14c2381/app/assets/stylesheets/components/forms.scss) file?